### ISSUE TITLE
Will there be a yolov8 paper released?I hope you don't keep eating hats！

### ISSUE THREAD

**Issue by WangRongsheng** [ultralytics/ultralytics#204]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Will there be a yolov8 paper released?I hope you don't keep eating hats.
hhh(●'◡'●)

**Comment by glenn-jocher** [comment]
Yes we're taking it seriously, we should have something soon, but so far I've only gotten through the abstract and intro.

**Comment by bakeryproducts** [comment]
@glenn-jocher  

> I've only gotten through the abstract and intro.

for v5 or v8?

**Comment by WangRongsheng** [comment]
> It is been said in TODO [README.md/known-issues--todos](https://github.com/ultralytics/ultralytics#known-issues--todos). I hope that answer your question.

thanks！

**Comment by WangRongsheng** [comment]
> Yes we're taking it seriously, we should have something soon, but so far I've only gotten through the abstract and intro.

Fighting！Good luck！

**Comment by WangRongsheng** [comment]
> @glenn-jocher
> 
> > I've only gotten through the abstract and intro.
> 
> for v5 or v8?

yes, it's v8！

**Comment by trohit920** [comment]
is there any update on this?

**Comment by glenn-jocher** [comment]
@trohit920 there is no new update on the release of a YOLOv8 paper. However, the development team is currently working on it and are hoping to release it soon. As with any scientific paper, it takes time and effort to ensure that it is comprehensive and accurate, so we appreciate your patience as we continue this process. In the meantime, if you have any questions about YOLOv8 or need assistance using it, please feel free to ask on the Ultralytics YOLOv8 GitHub Issues or visit https://docs.ultralytics.com for more information.

**Comment by h-fernand** [comment]
Have there been any updates on this? It's been a year since the issue was originally raised.

**Comment by glenn-jocher** [comment]
@h-fernand no updates on the YOLOv8 paper yet. We're focused on improving the codebase and documentation at https://docs.ultralytics.com. Thanks for your patience! 🚀

**Comment by achuntelolan** [comment]
Is it still not released the YOLOv8 paper ?

**Comment by glenn-jocher** [comment]
@achuntelolan we're still working on the YOLOv8 paper and aiming to share it with the community as soon as it's ready. Your enthusiasm is much appreciated! Keep an eye on our GitHub repo for updates. 😊🚀

**Comment by Blagues** [comment]
Hey, is there any update on this? And/or will there be a paper for Yolo11?

**Comment by glenn-jocher** [comment]
There are no updates on the YOLOv8 paper yet, and as for YOLO11, a paper is not currently planned. Please stay tuned to our GitHub for any future announcements.
